File: docs/dev_docs/nicepay/NICEPAY-IOS/NICEPAY-IOS/objc/NicePayAppSample/NicePayAppSample/Sample/RootViewController.m
//
//  RootViewController.m
//  NicePayAppSample
//
//  Created by xoghzone on 2019. 1. 9..
//  Copyright © 2019년 Nicepayments. All rights reserved.
//

#import "RootViewController.h"
#import "PaymentWebViewController.h"

@interface RootViewController ()

@end

@implementation RootViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    self.title = @"초기화면";
    
    self.verNo.text = [NSString stringWithFormat:@"Ver %@",
                       [[[NSBundle bundleForClass:[self class]] infoDictionary] objectForKey:@"CFBundleVersion"]];
}

- (IBAction)requestPayment:(id)sender {
    PaymentWebViewController *vc = [[PaymentWebViewController alloc] init];
    [self.navigationController pushViewController:vc animated:YES];
    [vc release];
}

@end

File: docs/dev_docs/nicepay/NICEPAY-IOS/NICEPAY-IOS/swift/NicepayAppSample/NicepayAppSample/Sample/RootViewController.swift
//
//  RootViewController.swift
//  NicepayAppSample
//
//  Created by xoghzone on 2019/11/27.
//  Copyright © 2019 Nicepayment. All rights reserved.
//

import UIKit

class RootViewController: UIViewController {
    @IBOutlet weak var verNo : UILabel?;
    
    override func viewDidLoad() {
        super.viewDidLoad()

        title = "초기화면"
        
        // Do any additional setup after loading the view.
        var version: String? {
            guard let dic = Bundle.main.infoDictionary,
                let v = dic["CFBundleVersion"] as? String
                else {
                    return nil
            }
            
            return v
        }
        
        verNo?.text = version ?? ""
    }
    
    @IBAction func requestPayment(sender:Any) {
        let vc = PaymentWebViewController()
        self.navigationController?.pushViewController(vc, animated: true)
    }
}
